R/summary-log.R

```diff
@@ -43,16 +43,8 @@
 #' @importFrom glue glue
 #' @importFrom tidyr unnest_wider
 #' @export
-summary_log <- function(
-  .base_dir = get_model_directory(),
-  .recurse = TRUE,
-  ...
-) {
-
-  # if no directory defined, set to working directory
```

Comment:
the only thing better than removing unnecessary code is removing unnecessary duplicated code. Well done.